README.md

```diff
@@ -112,6 +112,33 @@ The format for a profile is below:
 }
 ```
 
+The format for an unstructured profile is below:
+```
+{
+    "global_stats": {
+        "samples_used": int,
+        "empty_line_count": int,
+        "file_type": string,
+        "encoding": string
+    },
+    "data_stats": {
+        "data_label": {
+            "entity_counts": {
+                "word_level": dict(int),
+                "true_char_level": dict(int),
+                "postprocess_char_level": dict(int)
+            },
+            "times": dict(float)
+        },
+        "statistics": {
+            "vocab": list(char),
+            "words": list(string),
```

Comment:
should note these are minus the stop words

Reply:
do we need a comment for stopwords here?